tour/authorizations-code.md

```diff
@@ -5,51 +5,51 @@ title: Authorizations Code
+static void exercise(MiniAccumuloCluster mac, AccumuloClient client) throws Exception {
+    // create a table called "GothamPD".
+    client.tableOperations().create("GothamPD");
 
     // Create a "secretId" authorization & visibility
     final String secretId = "secretId";
     Authorizations auths = new Authorizations(secretId);
     ColumnVisibility colVis = new ColumnVisibility(secretId);
 
     // Create a user with the "secretId" authorization and grant him read permissions on our table
-    conn.securityOperations().createLocalUser("commissioner", new PasswordToken("gordonrocks"));
-    conn.securityOperations().changeUserAuthorizations("commissioner", auths);
-    conn.securityOperations().grantTablePermission("commissioner", "GothamPD", TablePermission.READ);
+    client.securityOperations().createLocalUser("commissioner", new PasswordToken("gordonrocks"));
+    client.securityOperations().changeUserAuthorizations("commissioner", auths);
+    client.securityOperations().grantTablePermission("commissioner", "GothamPD", TablePermission.READ);
 
     // Create 3 Mutation objects, securing the proper columns.
     Mutation mutation1 = new Mutation("id0001");
-    mutation1.put("hero","alias", "Batman");
-    mutation1.put("hero","name", colVis, "Bruce Wayne");
-    mutation1.put("hero","wearsCape?", "true");
+    mutation1.put("hero", "alias", "Batman");
+    mutation1.put("hero", "name", colVis, "Bruce Wayne");
+    mutation1.put("hero", "wearsCape?", "true");
     Mutation mutation2 = new Mutation("id0002");
-    mutation2.put("hero","alias", "Robin");
-    mutation2.put("hero","name", colVis,"Dick Grayson");
-    mutation2.put("hero","wearsCape?", "true");
+    mutation2.put("hero", "alias", "Robin");
+    mutation2.put("hero", "name", colVis, "Dick Grayson");
+    mutation2.put("hero", "wearsCape?", "true");
     Mutation mutation3 = new Mutation("id0003");
-    mutation3.put("villain","alias", "Joker");
-    mutation3.put("villain","name", "Unknown");
-    mutation3.put("villain","wearsCape?", "false");
+    mutation3.put("villain", "alias", "Joker");
+    mutation3.put("villain", "name", "Unknown");
+    mutation3.put("villain", "wearsCape?", "false");
 
     // Create a BatchWriter to the GothamPD table and add your mutations to it.
     // Once the BatchWriter is closed by the try w/ resources, data will be available to scans.
-    try (BatchWriter writer = conn.createBatchWriter("GothamPD", new BatchWriterConfig())) {
+    try (BatchWriter writer = client.createBatchWriter("GothamPD", new BatchWriterConfig())) {
         writer.addMutation(mutation1);
         writer.addMutation(mutation2);
         writer.addMutation(mutation3);
     }
 
     // Read and print all rows of the commissioner can see. Pass Scanner proper authorizations
-    Connector commishConn = mac.getConnector("commissioner", "gordonrocks");
+    AccumuloClient commishConn = mac.createAccumuloClient("commissioner", new PasswordToken("gordonrocks"));
```

Comment:
Yeah but then the tour would require a new developer to setup a running instance of Accumulo if they don't have one.

Reply:
> Yeah but then the tour would require a new developer to setup a running instance of Accumulo if they don't have one.

Not suggesting someone should have to do that for the tour.  Just thinking it would be better if the code in the tour worked on mini and a real instance, instead of just mini.

Reply:
> > Yeah but then the tour would require a new developer to setup a running instance of Accumulo if they don't have one.
> 
> Not suggesting someone should have to do that for the tour. Just thinking it would be better if the code in the tour worked on mini and a real instance, instead of just mini.

Ah OK I see what you are suggesting now.

Reply:
The change in cfc2081 is really nice, so much less clunky than what I proposed.